packages/space-search/space-search.0.9/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+name: "space-search"
+version: "0.9"
+maintainer: "Konstantin Weitz <konstantin.weitz@gmail.com>"
+dev-repo: "https://github.com/konne88/SpaceSearch"
```

Comment:
Could you please change this to
`dev-repo: "https://github.com/konne88/SpaceSearch.git"`

and add 
`bug-reports: "https://github.com/konne88/SpaceSearch/issues"`

?